### DISCUSSION TITLE
[Bug]: I am unable to configure s3 to pimcore

### DISCUSSION THREAD

**Discussion by pimcoreanand** [pimcore/pimcore#16579] [Q&A / Support]
### Pimcore version

v11.1.4

### Steps to reproduce

I have installed pimcore 11 and configured in ubuntu and want to store files and folders in s3 instead local

### Actual Behavior

I have installed pimcore 11 and configured in ubuntu and want to store files and folders in s3 instead local

### Expected Behavior

I have installed pimcore 11 and configured in ubuntu and want to store files and folders in s3 instead local

**Comment by pimcoreanand** [comment]
Kindly help me to setup pimcore assets to s3

**Comment by kingjia90** [comment]
Please have a look into https://pimcore.com/docs/platform/Pimcore/Installation_and_Upgrade/System_Setup_and_Hosting/File_Storage_Setup/#example-aws-s3-adapter-for-assets

If you have any specific blocker, please share more details

**Reply by pimcoreanand** [reply]
After configured AWS S3 Adapter for Assets,, 

I followed below steps.
composer require league/flysystem-aws-s3-v3:^2.0

# config/packages/prod/flysystem.yaml
services:
    assets_s3:
        class: 'Aws\S3\S3Client'
        arguments:
            -  endpoint: 'https://s3.eu-central-1.amazonaws.com'
               region: 'eu-central-1'
               version: 'latest'
               credentials:
                   key: '%env(S3_STORAGE_KEY)%'
                   secret: '%env(S3_STORAGE_SECRET)%'
                   
                   
# config/packages/prod/flysystem.yaml
flysystem:
    storages:
        pimcore.asset.storage:
            adapter: 'aws'
            visibility: public
            options:
                client: 'assets_s3'
                bucket: 'bucket-name'
                prefix: assets
                
 What is the next step to do in Ubuntu system
